src/components/AvatarEdit.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import { View, StyleProp, ViewStyle } from "react-native";
+import Icon from "./Icon";
+import Touchable from "./Touchable";
+import Avatar from "./Avatar";
+import { withTheme } from "../core/theming";
+
+import { GROUPS, COMPONENT_TYPES, FORM_TYPES } from "../core/component-types";
+import theme from "../styles/DefaultTheme";
+
+interface Props {
+  image: string | Blob;
```

Comment:
does this work? do you need to import the Blob from somewhere like web api?

Reply:
It works. Blob was known by default no need for any imports. It's bundled in with javascript I guess.